extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -441,7 +458,7 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
       Number(this.element.getAttribute('width'));
     const height = Number(this.element.getAttribute('data-override-height')) ||
       Number(this.element.getAttribute('height'));
-    this.initialSize_ = width && height
+    this.initialSize_ = this.isFluid_ ? {width: 0, height: 0} : width && height
```

Comment:
Add some parens to make sure execution order is well documented

Reply:
Done.